docker/ravendb-ubuntu/create-database.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+function putdb() {
+    
+    function usage() {
+        echo "Putdb creates database on Raven.Server via HTTP(S) request, that is named after RAVEN_DATABASE environmental variable."
+        echo "It should take no arguments when working with unsecured server."
+        echo "In the case of secured server it should take path to .crt certificate file as first arg and path to .key file as second to work properly."
+        echo "It returns 1 if cURL fails and 2 if there's wrong input."
+    }
+    [[ $# -eq 0 ]] && mode=unsecured || mode=secured
+    if [[ $mode == unsecured ]]; then
+        if curl -s $serverUrl'/admin/databases?name='$RAVEN_DATABASE -X PUT --compressed --data-raw '{"DatabaseName":"'$RAVEN_DATABASE'"}' > /dev/null; then 
+        echo "Database $RAVEN_DATABASE created successfully on unsecured server."
+        else
+        echo "Database $RAVEN_DATABASE wasn't created successfully - see error output above for details."
+        exit 1
+        fi
+    else
+        if [[ $mode == secured ]]; then
+            public_uri=$(./get-server-var.sh Configuration.Core.PublicServerUrl.UriValue)
+            public_tcp_uri=$(./get-server-var.sh Configuration.Core.PublicTcpServerUrl.UriValue)
+            if curl -s "$serverUrl/admin/databases?name=$RAVEN_DATABASE" -X PUT -k \
+                --cert $1 \
+                --key $2 \
+                --resolve $public_uri:127.0.0.1 \
```

Comment:
this 127 here might be problematic sometimes - it is going to only work if we're using `0.0.0.0` or `127.0.0.1` in the container. If somebody explicitly binds to e.g. container's internal address then we won't be able to reach it. I think we should only use `--resolve` when `ServerUrl` is either `0.0.0.0` or `127.0.0.1` - in other cases we should just try using whatever DNS gives us.

Reply:
applied the fix in latest commit